### PR TITLE
Added Contact Page [Issue #19]

### PR DESCRIPTION
**Pull request by roysammy123**
I have added the contacts page and linked it to the button in navbar as per [issue #19](https://github.com/subhadeeproy3902/BloxAI/issues/19) which was assigned to me. Have used shadcn ui components as instructed.

Here are screenshots of the Contact page:
<br>
Screenshot 2024-05-14 022559
Screenshot 2024-05-14 022654

<br><br>
Please review and merge @subhadeeproy3902 , @Xeven777

### CODE REVIEW

**Comment by subhadeeproy3902** [comment]
@roysammy123 Check whatsapp

File: src/app/contact/layout.tsx
Comment (Xeven777) [inline src/app/contact/layout.tsx:1]: Great work @roysammy123 . i would request a small change here:
For the Metadata, change the title to : " BloxAI | Contact us " and change the description too accordingly ☺️

File: src/app/globals.css [hunk 1]
```diff
@@ -1,78 +1,76 @@
...
+
+      --destructive: 0 62.8% 30.6%;
+      --destructive-foreground: 60 9.1% 97.8%;
+
+      --border: 12 6.5% 15.1%;
+      --input: 12 6.5% 15.1%;
+      --ring: 24 5.7% 82.9%;
+    }
   }
-}
```
Comment (Xeven777) [inline src/app/globals.css:66]: would u please revert back these chnages in globals.css too .

**Review by Xeven777 — CHANGES_REQUESTED** [review]
Please, do the changes i requested and we'll be happy to merge the PR😁

**Comment by roysammy123** [comment]
Have made the changes. Please check and let me know if I need to do anything else @Xeven777 @subhadeeproy3902

**Pull request closed without merging**